optuna/visualization/parallel_coordinate.py

```diff
@@ -80,6 +80,12 @@ def _get_parallel_coordinate_plot(study, params=None):
         all_params = set(params)
     sorted_params = sorted(list(all_params))
 
+    labelangle = None
+    for p_name in sorted_params:
+        if len(p_name) > 10:
+            labelangle = 30
+            break
```

Comment:
Maybe a bit more Pythonic.
```python
if any(len(p_name) > 10 for p_name in sorted_params):
    labelangle = 30  
else: 
    labelangle = None
```
You can one-line it too but I'm skeptical it aids readability.